module/metrics/access.go

```diff
@@ -66,8 +88,26 @@ func NewAccessCollector() *AccessCollector {
 			Subsystem: subsystemConnectionPool,
 			Help:      "counter for the number of times a cached connection is evicted from the connection pool",
 		}),
+		lastFullBlockHeight: promauto.NewGauge(prometheus.GaugeOpts{
+			Name:      "last_full_block_height",
+			Namespace: namespaceAccess,
+			Subsystem: subsystemIngestion,
+			Help:      "gauge to track the highest consecutive height with all collections indexed",
```

Comment:
is this also a finalized block?

Reply:
Yes, collections are indexed after the block is finalized

Reply:
what about renaming to `last_full_finalized_block_height` to make it clear that it's finalized height.

whereas keep `max_receipt_height` as is, because when we receive the receipt from EN, it might not be finalized, am I right, or not?

Reply:
yea, I'll rename it.

the `max_receipt_height` metric is only updated after the block is finalized, but it may be received earlier:
https://github.com/onflow/flow-go/blob/9361a1cd7128b67d5a142e559f28c55796cb5d08/engine/access/ingestion/engine.go#L474-L477